test/links.ts

```diff
@@ -61,7 +61,7 @@ describe('link fields', function() {
                 context.connection.close();
                 done();
             });
-            var c = container.connect(listener.address());
+            var c = container.connect(listener.address() as any);
```

Comment:
As per the type definitions, connect takes an object of type ConnectionOptions whereas we are passing an object of type AddressInfo. Casting it to `any` fixes the problems.